crates/provider-blobstore-fs/src/lib.rs

```diff
@@ -258,13 +258,22 @@ impl Blobstore for FsProvider {
                 result_subject,
                 async {
                     let path = self.get_container(context, container).await?;
-                    let md = fs::metadata(path)
+                    let md = fs::metadata(&path)
                         .await
                         .context("failed to lookup directory metadata")?;
-                    let created_at = md.created().context("failed to lookup creation date")?;
-                    let created_at = created_at
-                        .duration_since(SystemTime::UNIX_EPOCH)
-                        .context("creation time before Unix epoch")?;
+
+                    let created_at = if let Ok(created_time) = md.created() {
+                        created_time
+                            .duration_since(SystemTime::UNIX_EPOCH)
+                            .context("creation time before Unix epoch")?
+                    } else {
+                        // NOTE: Some platforms don't have support for creation time, so we default to the unix epoch
+                        debug!(
```

Comment:
nit; can we get the error printed here as well, otherwise this is not too helpful? (probably want to use a `match`)

Reply:
Oh, yes